assets/js/components/DataBlock.js

```diff
@@ -128,12 +128,23 @@ class DataBlock extends Component {
 						'googlesitekit-data-block--is-gathering-data': gatheringData,
 					}
 				) }
-				tabIndex={ 'button' === context ? '0' : '-1' }
+				tabIndex={
```

Comment:
This change is adding a lot of inline conditionals, it might be clearer to group them into a single conditional, something like this:

```js
		const interactionProps = gatheringData
			? {
					tabIndex: '-1',
					'aria-disabled': true,
			  }
			: {
					tabIndex: 'button' === context ? '0' : '-1',
					onClick: handleStatSelection && this.handleClick,
					onKeyPress: handleStatSelection && this.handleKeyPress,
			  };


		return (
			<div
				// ...
				{ ...interactionProps }
			>
```

Obviously, removing those props from the inline JSX. What do you think?

Reply:
@techanvil Not sure about that. I prefer to keep it simple like it is currently and it's easier to read 🤷

Reply:
Ok, no worries - was just a suggestion, happy to keep it as it is. :+1: